### DISCUSSION TITLE
Help: ComplexData for Non-Model Organism

### DISCUSSION THREAD

**Discussion by HossFir** [SysBioChalmers/GECKO#366] [GECKO3]
In case there is no available data for protein complexes in (https://www.ebi.ac.uk/complexportal/complex/organisms), would it be acceptable to proceed with updating the `rxnEnzMat` by incorporating the corresponding protein copy numbers (represented as stoichiometry in `rxnEnzMat`), so the stoichiometry for the targeted proteins updated? 

For example, the copy number of this protein `P54205` is 8, which originally had a default stoichiometry of 1 in `rxnEnzMat`. I updated the stoichiometry for this protein across the corresponding reactions to 8 in `rxnEnzMat`.

**Comment by edkerk** [comment]
I'm not sure I get what you exactly mean with "the copy number of this protein `P54205` is 8", do you mean that it forms a homooctamer?

**Reply by HossFir** [reply]
Yes, I updated the subunits of those proteins with more than one subunit. So, as an example, `P54205 `has eight subunits, so its stoichiometry would be 8 in `rxnEnzMat`. Or this `P73617`, a Homodimer, has a stoichiometry of two.

**Reply by edkerk** [reply]
Consider that the enzyme usage is defined as (MW/kcat)*stochiometry, increasing the subunit number would increase the enzyme usage to catalyze the same amount of reaction. It assumes that the complex consists of multiple subunits, but that there is still one active site (the kcat remains unchanged). This might or might not be realistic.

**Reply by edkerk** [reply]
This is particularly relevant for homomers, as there it is more likely that each subunit contributes with an active site. Heteromers might also still have multiple active sites, and strictly speaking we should have a correction for this, but I'm not aware where this information could be gathered from (not from Complex Portal it seems). I had a quick look through Complex Portal for _S. cerevisiae_ and it does not seem to include any homomers.

So my advice would be not to manually consider stochiometry of homomers.

**Reply by HossFir** [reply]
Thanks for your reply. 
I curated the UniProt data so that if the proteins were not considered monomers, I updated their stoichiometry. So, if the protein is Homodimer, it has a stoichiometry of 2, Homotrimer is 3, and so for the rest (I did it for both Hetero and Homo). 
I simulated the GECKO model, and with the proteomics data integrated into the model, the minimum protein usage was mostly the same with the model equipped with stoichiometry and the one that did not.
